go.mod

```diff
@@ -35,6 +35,7 @@ require (
 )
 
 replace (
+	github.com/hashicorp/hcat => ../hashicat
```

Comment:
Tests are failing looking for your replace directive for locally

Reply:
Sigh. Fixed.